TF_Settings_Web/src/TouchFree/Configuration/ConfigurationManager.ts

```diff
@@ -0,0 +1,101 @@
+import {
```

Comment:
Is this a copy of the source for tooling to add it to the settings UI? 

I know we've talked about automating the maintaining of this but just wanted to check

Reply:
Yes, the `src/TouchFree` directory is a clone of the Web Tooling for now (until we do the work you mentioned). I've made sure that the two are identical as of the last commit I made but this is definitely another motivator to _either_ do the automatic import, to work out what the workflow for people who want to consume tooling as TypeScript looks like, or possibly both?

Reply:
I think both are important going forward. This is fine for the time being though